src/link/main.c

```diff
@@ -126,7 +127,14 @@ FILE *openFile(char const *fileName, char const *mode)
 	if (!fileName)
 		return NULL;
 
-	FILE *file = fopen(fileName, mode);
+	FILE *file;
+	if (strcmp(fileName, "-") != 0) {
+		file = fopen(fileName, mode);
+	} else if (strchr(mode, 'r')) {
```

Comment:
Given that `r` has to be the first character, it'd be better to just check that.

Also, in theory, the braces don't match our code standard, but I guess a `clang-format` *really* is needed...

Reply:
I'm not seeing the code style mismatch? The other point you are right, didn't know that.

Reference: https://github.com/gbdev/rgbds/blob/master/src/asm/lexer.c#L977

Reply:
Since our style is a mess anyways (#612), I don't really care.

Reply:
The braces/no braces thing is pretty annoying in the case of the Linux style. Having braces is usually better, as it prevents problems like: https://blog.codecentric.de/en/2014/02/curly-braces/

Reply:
Oh that way having braces for single statements. I usually put them in as soon as I have an else.

But this was also a case where I initially had some logging lines in there to make sure it worked.